src/main/java/io/jenkins/plugins/sprp/models/Step.java

```diff
@@ -0,0 +1,57 @@
+package io.jenkins.plugins.sprp.models;
+
+import java.util.AbstractMap;
+import java.util.HashMap;
+
+public class Step {
+    private String stepName;
+    private String defaultParameter;
+    private HashMap<String, Object> parameters;
+
+    Step(){
+
+    }
+
+    Step(String stepName){
+        System.out.println("stepname: " + stepName);
+        if(!stepName.contains(" "))
+            this.stepName = stepName;
+        else{
+            String[] str = stepName.split(" ", 0);
+            this.stepName = str[0];
+            this.defaultParameter = str[1];
+
+            if(this.defaultParameter.startsWith("'") || this.defaultParameter.startsWith("\""))
+                this.defaultParameter = this.defaultParameter.substring(1, this.defaultParameter.length() - 1);
```

Comment:
I think you are trying to get the second word in a list of whitespace separated words. If there are multiple whitespaces separating the first word from the other words, I think this code fails, e.g. `first   second third` fails because there are three spaces between `first` and `second`.

Why not use the yaml object model to parse the step parameters?

Reply:
I want a format similar to ```sh "parameter"```. I was not able to make it work using yaml object model.